python_examples/face_recognition.py

```diff
@@ -114,6 +114,28 @@
         # In particular, a padding of 0.5 would double the width of the cropped area, a value of 1.
         # would triple it, and so forth.
 
+        # There is another overload of compute_face_descriptor that can take
+        # as an input aligned image of size 150x150. 
+        #
+        # Here is a sample usage of that
+
+        print("Computing descriptor on aligned image ..")
+        
+        # We will first generate the aligned image
+        face_chip = dlib.get_face_chip(img, shape)        
+
+        assert face_chip.shape[0] == face_chip.shape[1]
```

Comment:
These asserts shouldn't be in this file.  This file is meant to be a tutorial, rather than unit test script.  It is however worth pointing out in a comment the invariants represented by these asserts.  Like that the input chip must be cropped the way `dlib.get_face_chip` crops faces.

Reply:
ok